### DISCUSSION TITLE
info.plist

### DISCUSSION THREAD

**Discussion by karokhaziz** [tofi86/universalJavaApplicationStub#114] [General]
hi thanks for your work man 
i need an info.plist example can you help me please
thanks

**Comment by tofi86** [comment]
You can download the free "pagina EPUB-Checker" Mac App from https://www.pagina.gmbh/produkte/epub-checker/#c773 and open with a right click "show package content" and navigate to the "Info.plist".
It's generated from this Maven build plugin: https://github.com/paginagmbh/EPUB-Checker/blob/master/pom.xml#L368-L403

This Info.plist file from the demo project is also worth a look: https://github.com/tofi86/universalJavaApplicationStubDemo/blob/master/appbundle/UniversalJavaApplicationStubDemo/UniversalJavaApplicationStubDemo.app/Contents/Info.plist

**Reply by karokhaziz** [reply]
thanks a lot for your help
